tests/semver/test_sem_ver_utils.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from poetry.semver import is_sem_ver_constraint
+from poetry.semver import sem_ver_sorted
+
+
+@pytest.mark.parametrize(
+    "constraint,result",
+    [
+        ("*", True),
+        ("*.*", True),
+        ("v*.*", True),
+        ("*.x.*", True),
+        ("x.X.x.*", True),
+        # ('!=1.0.0', Constraint('!=', '1.0.0.0')),
```

Comment:
whats going on here?

Reply:
- adapted from `tests/semver/test_main.py`
- enabled and passing in amended commit (to follow)